TypeCobol.Analysis.Test/BasicCfgInstrs/PerformProcIterativeAfterRecursive0.dot

```diff
@@ -0,0 +1,78 @@
+﻿digraph Cfg {
+compound=true;
+node [
+shape = "record"
+]
+
+edge [
+arrowtail = "empty"
+]
+Block0 [
+label = "{START|}"
+]
+Block10 [
+label = "{PARAREC. Block10|    add 1 to n\l    display n\l    if n \< 3\l}"
+]
+Block11 [
+label = "{Block11|       display 'before perform'\l}"
+]
+Block12 [
+label = "{Block12|       perform pararec  varying n from 1 by 1 until n \> 5\l}"
+]
+subgraph cluster_2{
+color = blue;
+label = "pararec";
+Block21 [
+label = "{PARAREC. Block21|    add 1 to n\l    display n\l    if n \< 3\l}"
+]
+Block22 [
+label = "{Block22|       display 'before perform'\l}"
+]
+Block24 [
+label = "{Block24|    else\l       display 'not done'\l}"
+]
+Block25 [
+label = "{Block25|    display 'terminate'\l}"
+]
+Block21 -> Block22
+Block21 -> Block24
+Block22 -> Block12
+Block24 -> Block25
+Block25 -> Block12
+
+}
+Block12 -> Block21 [style=dashed]
+
+Block13 [
+label = "{Block13|       display 'after perform'\l}"
+]
+Block15 [
+label = "{Block15|    display 'terminate'\l}"
+]
+Block1 [
+label = "{Block1|    perform pararec test after varying n from 1 by 1 until n \> 3\l}"
+]
+subgraph cluster_1{
+color = blue;
+label = "pararec";
+
+}
+Block1 -> Block10 [style=dashed]
+
+Block2 [
+label = "{Block2|    goback\l}"
+]
+Block14 [
+label = "{Block14|    else\l       display 'not done'\l}"
+]
+Block0 -> Block10
```

Comment:
Here Block0 should point to block1 `perform pararec test after varying n from 1 by 1 until n > 3`.

Reply:
So after our discussion, this should be adressed through a separate issue (#1783)